Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -110,7 +113,32 @@ - (dispatch_queue_t)methodQueue
     };
   }
 
-  [ctrl presentViewController:share animated:YES completion:nil];
+  /*
+   * The `anchor` option takes a view to set as the anchor for the share
+   * popup to point to, on iPads running iOS 8. If it is not passed, it
+   * defaults to centering the share popup on screen without any arrows.
+   */
+  if ([share respondsToSelector:@selector(popoverPresentationController)]) {
+    NSNumber *anchorViewTag = [RCTConvert NSNumber:options[@"anchor"]];
+    if (anchorViewTag) {
+      dispatch_async(self.bridge.uiManager.methodQueue, ^{
```

Comment:
This method is already being executed on the main thread, no need to dispatch_async.